### PR TITLE
bug fix when only one factor value is provided

### PR DESCRIPTION
**Pull request by ypriverol**

### CODE REVIEW

File: sdrf_pipelines/openms/openms.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def openms_convert(self, sdrf_file: str = None, keep_raw: bool = False, one_tabl
         if not split_by_columns:
             # get factor columns (except constant ones)
             factor_cols = [c for ind, c in enumerate(sdrf) if
-                           c.startswith('factor value[') and len(sdrf[c].unique()) > 1]
+                           c.startswith('factor value[') and len(sdrf[c].unique()) >= 1]
```
Comment (jpfeuffer) [inline sdrf_pipelines/openms/openms.py:137]: Can you adapt the comment accordingly?

File: sdrf_pipelines/openms/openms.py [hunk 2]
```diff
@@ -134,7 +134,7 @@ def openms_convert(self, sdrf_file: str = None, keep_raw: bool = False, one_tabl
...
 
             # get characteristics columns (except constant ones)
```
Comment (jpfeuffer) [inline sdrf_pipelines/openms/openms.py:139]: What about "characteristics"? Do you want to do the same here?

Comment (ypriverol) [reply sdrf_pipelines/openms/openms.py:139]: Yes it should be the same!!!

File: sdrf_pipelines/openms/openms.py (hunk 1 above)
Comment (jpfeuffer) [reply sdrf_pipelines/openms/openms.py:137]: I.e. remove the "(except constant)" in both cases. Here and at characteristics.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**